larky/src/main/resources/vgs/nts.star

```diff
@@ -1,18 +1,20 @@
 load("@vgs//native_nts", _nts="native_nts")
+load("@vgs//vault", vault="vault")
 load("@stdlib//larky", larky="larky")
 load("@vendor//jsonpath_ng", jsonpath_ng="jsonpath_ng")
 
 
 def render(
-    input,
-    pan,
-    cvv,
-    amount,
-    currency_code,
-    exp_month=None,
-    exp_year=None,
-    cryptogram_value=None,
-    cryptogram_eci=None,
+        input,
+        pan,
+        cvv,
+        amount,
+        currency_code,
+        dcvv=None,
+        exp_month=None,
+        exp_year=None,
+        cryptogram_value=None,
+        cryptogram_eci=None,

```

Comment:
Code style. We should use the same indentation in the code base, or at least for the same file. 
A good way to find out what's changed intentionally or not is to review your own code.
Auto code formatter might also help.

Reply:
please yes! run it as a ci check to make sure everything's linted 🙏 🙌